src/main/resources/seamless_loading_screen.mixins.json

```diff
@@ -10,7 +10,6 @@
     "GameRendererMixin",
     "LevelLoadingScreenMixin",
     "LevelStorageMixin",
-    "MinecraftClientMixin",
```

Comment:
This does not seem correct.. Is not the mixin is still needed? And if it is not, why not remove it?